assignment2.html

```diff
@@ -0,0 +1,89 @@
+<html>
+  <head><link rel="stylesheet" style="text\css" href="assignment2.css"></head>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Pleace this css file at loacl storage level, don't give reference from external url.